### PR TITLE
Feature/icons

### PR DESCRIPTION
**Pull request by RishikeshVedpathak**
Added four new icons - email-outline, phone, search and whatsapp.

### CODE REVIEW

File: packages/components/src/icons/EmailOutline.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, { SVGProps } from "react";
+
+const SvgEmailOutline = (props: SVGProps<SVGSVGElement>) => (
+  <svg width={24} height={24} {...props}>
+    <defs>
```
Comment (kushalmahajan) [inline packages/components/src/icons/EmailOutline.tsx]: Refer the shape of other icons and align your SVG markup to that for all icons.

File: packages/components/src/icons/Phone.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React, { SVGProps } from "react";
+
+const SvgPhone = (props: SVGProps<SVGSVGElement>) => (
+  <svg width={17} height={16} {...props}>
```
Comment (kushalmahajan) [inline packages/components/src/icons/Phone.tsx]: Default width and height is 24 x 24. If designers have provided something else, you need to get it fixed or fix yourself.

Comment (RishikeshVedpathak) [reply packages/components/src/icons/Phone.tsx]: changes are done. please check updated PR.

Comment (kushalmahajan) [reply packages/components/src/icons/Phone.tsx]: The icons are not rendered correctly. You may want to run the generate-icons script.

**Pull request merged**